views/js/assets/strategies.js

```diff
@@ -54,6 +54,20 @@ define([
      */
     var strategies = {
 
+        externalReplaced : {
+            name : 'externalReplaced',
+            handle : function handlePackedUrl(url, data){
+                if(!_.isUndefined(url.source) && !_.isUndefined(data.assets)) {
+                    for (var key in data.assets){
```

Comment:
this violoates the ESLint rules "variable declaration should reflect hoisting", did you have configured your IDE ?